driver/node/opera.go

```diff
@@ -207,7 +207,18 @@ func (n *OperaNode) StreamLog() (io.ReadCloser, error) {
 }
 
 func (n *OperaNode) Stop() error {
-	return n.host.Stop()
+	// SigInt repeatedly up to 9 times
+	if err := network.Retry(9, 5*time.Second, func() error {
```

Comment:
this code does not look right. 
* first, I think calling `Retry` in this context only sends the signal to the container 9 times, but it does not actually assures that the container has acted. I.e. I think it is not necessary to send the signal in a loop. 
* It check if `err == nil` - I guess it was supposed to be `err !=nil` 

It would make sense to somehow wait until the container is down, but it would have to be done differently. 

the originally called method `host.Stop()` already does what is implemented here - it sends signal to the container to gracefully stop and kills the container only after certain timeout. If it does not work, I think the right way is to investigate what is wrong in current mechanism.  It may be well that Sonic does not properly react to the signal, but then  sending extra signals here is only a workaround.

Reply:
Change removed.

Reply:
I did some background reading, and it seems that this is the problem: https://docs.docker.com/reference/cli/docker/container/kill/

i.e. signals are not propagated to processes spawn from `bash`, i.e., does not matter how many signals is sent here, it will not work.

Reply:
> With the -g option, Tini kills the child process group , so that every process in the group gets the signal. This corresponds more closely to what happens when you do ctrl-C etc. in a terminal: The signal is sent to the foreground process group.

`tini -g` was what I was going for.